module/src/rotation.js

```diff
@@ -138,7 +138,7 @@ async function rotateViaRotation(deltaX, deltaY, document, update){
 
 
 async function rotateTokenOnPreUpdate(token_document, change, options, userId) {
-    const token_data = token_document.data
+    const token_data = token_document
```

Comment:
Also, two things about `module.json`: 
1. Versions should be updated to include support for v10.
2. Should the module continue to support v8?